src/mapd-client-v2.js

```diff
@@ -11,8 +11,8 @@ export default function MapDClientV2(protocol) {
 MapDClientV2.prototype = Object.create(MapDClient.prototype)
 
 /* eslint-disable no-unused-expressions */
-!(function() {
-  [
+!(function () {
+  ;[
```

Comment:
What's up with the `;`?

Reply:
I'm assuming that's to keep the linter happy. I've seen something similar with an added semicolon prevents the mistaken detection of an undefined return value, e.g. https://stackoverflow.com/questions/53013437/expected-assignment-or-function-call-no-unused-expressions-reactjs

Reply:
This was automatically added by eslint. I believe it is this rule https://eslint.org/docs/rules/no-unexpected-multiline